### PR TITLE
Iitc tool as new tag

### PR DESCRIPTION
**Pull request by McBen**
Replace `@something@ ` build tags by something that look and feels like regular function calls.

old -> `'@include_raw:external/leaflet-src.js@';`
new -> `IITCTool.import('external/leaflet-src.js');`

old -> `iconUrl: '@include_img:images/marker-ingress.png@',`
new -> `iconUrl: IITCTool.importImage('images/marker-ingress.png'),`
( `include_string` looks the same)

with some extra code for css:
old ->`$('<style>').prop('type', 'text/css').html('@include_css:bookmarks.css@').appendTo('head');`
new -> `IITCTool.importCSS('bookmarks.css');`

### CODE REVIEW

File: core/total-conversion-build.js [hunk 1]
```diff
@@ -49,11 +46,12 @@ if (!window.PLAYER || !PLAYER.nickname) {
...
 document.head.innerHTML = ''
   + '<title>Ingress Intel Map</title>'
-  + '<style>'+'@include_string:style.css@'+'</style>'
-  + '<style>'+'@include_css:external/leaflet.css@'+'</style>'
-//note: smartphone.css injection moved into code/smartphone.js
+  + '<style>' + IITCTool.importString('style.css') + '</style>'
+  // note: smartphone.css injection moved into code/smartphone.js
   + '<link rel="stylesheet" type="text/css" href="//fonts.googleapis.com/css?family=Roboto:100,100italic,300,300italic,400,400italic,500,500italic,700,700italic&subset=latin,cyrillic-ext,greek-ext,greek,vietnamese,latin-ext,cyrillic"/>';
 
+IITCTool.importCSS('external/leaflet.css');
```
Comment (le-jeu) [inline core/total-conversion-build.js:53]: any reason to differenciate the main style and leaflet.css ?

File: plugins/distance-to-portal.js [hunk 2]
```diff
@@ -106,19 +106,19 @@ window.plugin.distanceToPortal.setupPortalsList = function() {
...
     window.plugin.distanceToPortal.currentLoc = L.latLng(JSON.parse(localStorage['plugin-distance-to-portal']));
-  } catch(e) {
+  } catch (e) {
     window.plugin.distanceToPortal.currentLoc = null;
   }
 
   window.plugin.distanceToPortal.currentLocMarker = null;
 
-  $('<style>').prop('type', 'text/css').html('@include_string:distance-to-portal.css@').appendTo('head');
+  $('<style>').prop('type', 'text/css').html(IITCTool.importString('distance-to-portal.css')).appendTo('head');
```
Comment (le-jeu) [inline plugins/distance-to-portal.js]: `importCSS`

File: plugins/bookmarks.js [hunk 3]
```diff
@@ -1363,7 +1355,8 @@ window.plugin.bookmarks.initMPE = function(){
     }
 
   }
+
 // moved setupCSS to the end to improve readability of built script
-    window.plugin.bookmarks.setupCSS = function() {
-    $('<style>').prop('type', 'text/css').html('@include_css:bookmarks.css@').appendTo('head');
-  }
+window.plugin.bookmarks.setupCSS = function () {
```
Comment (le-jeu) [inline plugins/bookmarks.js]: why not remove this function like ou did for the `keys` plugin ?

File: plugins/bookmarks.js [hunk 4]
```diff
@@ -501,32 +501,23 @@ window.plugin.bookmarks.loadStorageBox = function() {
...
+  });
 
-    $.each(plugin.bookmarks.bkmrksObj.portals, function(id, folder) {
-      $.each(folder.bkmrk, function(id, bookmark) {
-        if(bookmark.label.toLowerCase().indexOf(term) === -1) return;
+    $.each(plugin.bookmarks.bkmrksObj.portals, function (id, folder) {
+      $.each(folder.bkmrk, function (id, bookmark) {
+        if (bookmark.label.toLowerCase().indexOf(term) === -1) return;
 
-        query.addResult({
```
Comment (le-jeu) [inline plugins/bookmarks.js:519]: somehing looks wrong here

File: core/total-conversion-build.js (hunk 1 above)
Comment (McBen) [reply core/total-conversion-build.js:53]: leaflet needs "url" parsing.

File: plugins/bookmarks.js (hunk 4 above)
Comment (McBen) [reply plugins/bookmarks.js:519]: fixed 23eb241

the style differences of bookmark.js made it hard to edit. I had to switch the editor.

File: core/code/smartphone.js [hunk 5]
```diff
@@ -24,7 +24,7 @@ window.runOnSmartphonesBeforeBoot = function() {
   // add smartphone stylesheet
   var style = document.createElement('style');
   style.type = 'text/css';
-  style.appendChild(document.createTextNode('@include_string:smartphone.css@'));
+  style.appendChild(document.createTextNode(IITCTool.importString('smartphone.css')));
   document.head.appendChild(style);
```
Comment (le-jeu) [inline core/code/smartphone.js]: `importCSS` ?

File: plugins/distance-to-portal.js (hunk 2 above)
Comment (McBen) [reply plugins/distance-to-portal.js]: cab0df2

File: core/total-conversion-build.js (hunk 1 above)
Comment (le-jeu) [reply core/total-conversion-build.js:53]: ok
`style.css` might also use `url` at some point tho
also the name matches this use case, importing css.

File: core/code/smartphone.js (hunk 5 above)
Comment (McBen) [reply core/code/smartphone.js]: and..fixed. thx, cb46034

**Pull request closed without merging**